src/components/store/types/Store.ts

```diff
@@ -0,0 +1,28 @@
+import { Action } from './Action';
```

Comment:
We name files in lowercase at other places, so I would suggest to rename `Store.ts` to `store.ts` as well as other files.